### PR TITLE
Implemented functionality for view details button

### PR DESCRIPTION
**Pull request by tsivkadi**

### CODE REVIEW

File: src/components/offer-card/offer-actions/OfferActions.tsx [hunk 1]
```diff
@@ -9,35 +10,25 @@ import TurnedInNot from '@mui/icons-material/TurnedInNot'
...
 
 const OfferActions: FC<OfferActionsProps> = ({
   id,
   price,
   isBookmarked,
-  onBookmarkClick,
-  buttonActions
+  onBookmarkClick
 }) => {
-  const buttons = buttonActions.map((elem, index) => (
```
Comment (Tolik170) [inline src/components/offer-card/offer-actions/OfferActions.tsx:30]: I think, you can do this without deleting this code, just change 'onclick' to 'to' and add component property to buttonActions

File: src/containers/find-offer/offer-card-square/OfferCardSquare.tsx [hunk 2]
```diff
@@ -98,7 +103,12 @@ const OfferCardSquare: FC<OfferCardSquareProps> = ({
       <AppButton sx={styles.sendMessageButton}>
         {t('common.labels.sendMessage')}
       </AppButton>
-      <AppButton fullWidth sx={styles.viewDetailsButton}>
+      <AppButton
```
Comment (MickhaelSh) [inline src/containers/find-offer/offer-card-square/OfferCardSquare.tsx:106]: this card should use actions as well as big card

File: src/containers/find-offer/offer-container/OfferContainer.tsx [hunk 3]
```diff
@@ -32,34 +35,39 @@ const OfferContainer: FC<OfferContainerProps> = ({
...
+        label: t('common.labels.sendMessage'),
+        handleClick: () => null
+      },
+      {
+        label: t('common.labels.viewDetails'),
+        handleClick: () => null,
+        buttonProps: {
+          component: Link,
+          to: `${authRoutes.offerDetails.path}/${el._id}`
+        }
```
Comment (Yurenko) [inline src/containers/find-offer/offer-container/OfferContainer.tsx:50]: Why do you use buttonProps if you have handleClick for button handle?
You can pass the link to handleClick

File: src/containers/find-offer/offer-card-square/OfferCardSquare.tsx (hunk 2 above)
Comment (tsivkadi) [reply src/containers/find-offer/offer-card-square/OfferCardSquare.tsx]: fixed

**Review by MickhaelSh — APPROVED** [review]

File: src/router/constants/authRoutes.ts [hunk 4]
```diff
@@ -5,5 +5,5 @@ export const authRoutes = {
     route: 'categories/subjects/findOffers',
     path: '/categories/subjects/findOffers'
   },
-  offerDetails: { route: 'offerDetails/:id', path: 'offerDetails/:id' }
+  offerDetails: { route: 'offerDetails/:id', path: '/offerDetails' }
```
[the remark this answers is not in the document]
Comment (Tolik170) [reply src/router/constants/authRoutes.ts:8]: yeah, it will be conflict, so in rebase one of us will have to change it

**Review by mxrcury — APPROVED** [review]

**Review by FryDay969 — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**